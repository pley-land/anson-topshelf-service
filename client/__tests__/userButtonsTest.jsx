import React from 'react';
import renderer from 'react-test-renderer';
import UserButtons from '../components/userButtons';

describe('Testing UserButtons', () => {
  it('should render properly', () => {
    const output = renderer.create(<UserButtons />).toJSON();
    expect(output).toMatchSnapshot();
  });
});
