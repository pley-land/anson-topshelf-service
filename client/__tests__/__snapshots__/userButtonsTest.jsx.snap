// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing UserButtons should render properly 1`] = `
<div
  className="content-user-buttons"
>
  <button
    className="reviewButton"
    type="button"
  >
    <i
      className="material-icons"
    >
      star
    </i>
    <span
      className="buttonText"
    >
      Write a Review
    </span>
  </button>
  <div
    className="spanDiv"
  >
    <button
      className="buttonSpans leftSpan"
      type="button"
    >
      <i
        className="material-icons"
      >
        photo_camera
      </i>
      <span
        className="buttonText"
      >
        Add Photo
      </span>
    </button>
    <button
      className="buttonSpans midSpan"
      type="button"
    >
      <i
        className="material-icons"
      >
        share
      </i>
      <span
        className="buttonText"
      >
        Share
      </span>
    </button>
    <button
      className="buttonSpans rightSpan"
      type="button"
    >
      <i
        className="material-icons"
      >
        save
      </i>
      <span
        className="buttonText"
      >
        Save
      </span>
    </button>
  </div>
</div>
`;
